src/sagemaker/model.py

```diff
@@ -954,11 +981,16 @@ def package_for_edge(
             job_name = f"packaging{self._compilation_job_name[11:]}"
         self._init_sagemaker_session_if_does_not_exist(None)
         s3_kms_key = resolve_value_from_config(
-            s3_kms_key, EDGE_PACKAGING_KMS_KEY_ID_PATH, sagemaker_session=self.sagemaker_session
+            s3_kms_key,
+            EDGE_PACKAGING_KMS_KEY_ID_PATH,
+            sagemaker_session=self.sagemaker_session,
         )
         role = resolve_value_from_config(
             role, EDGE_PACKAGING_ROLE_ARN_PATH, sagemaker_session=self.sagemaker_session
         )
+        resource_key = resolve_value_from_config(
+            resource_key, EDGE_PACKAGING_RESOURCE_KEY_PATH, sagemaker_session=self
+        )
```

Comment:
Can you double check if `test_model.py` file also have any unit-tests to verify this change?

Reply:
Will double check, I couldn't find any test in test_model that verifies  the edge packaging attributes , I'll add one if not! Thanks

Reply:
Added a new UT testing the package for edge config injection. Thanks for this comment